cpp/src/groupby/common/aggregation_requests.cpp

```diff
@@ -52,7 +51,7 @@ std::vector<SimpleAggRequestCounter> compound_to_simple(
         if (op == MEAN) {
           columns_to_ops[col].insert(COUNT);
           columns_to_ops[col].insert(SUM);
-        } else {
+        } else if (op != MEDIAN && op != QUANTILE) { // Skip Median and Quantiles
```

Comment:
We should define some trait/function that abstracts such explicit logic checks. Otherwise, this function will become an entangled logic check mess.